website/docs/patterns/filter-patterns/partials/guidelines/guidelines.md

```diff
@@ -0,0 +1,134 @@
+## Structure
+
+A filtering pattern consists of a number of elements and Helios components that work together functionally and enforce a consistent layout. These elements are variable depending on the content of the data set and the context within the application, and can be combined in different ways to achieve an experience that benefits the end-user.
+
+### Data set
+
+A data set is a broad term for an array of items, objects, or related information presented as separate but related records. A data set is commonly represented via a [Table](/components/table) but may be expressed in other formats depending on the type of information.
+
+**Dataset represented by a table**
+
+![Tabular data set](/assets/patterns/filter-patterns/example-data-set-tabular-data.png)
+
+**Dataset represented by a grid**
+
+![Non-tabular data set](/assets/patterns/filter-patterns/example-data-set-non-tabular-data.png)
+
+### Filter positioning
+
+Filters are responsible for the functional aspect of limiting a data set and for displaying the [parameters](/patterns/filter-patterns?tab=core%20concepts#parameter) which a user can filter upon. They often consist of numerous parameters that mirror the columns of a table or metadata contained within the data set.
+
+Filter positioning is largely dependent on the complexity of the data set and the scope for which the filters are being applied (are the filters applied at the page level, or only to the data set they are paired with).
```

Comment:
When I ran a grammar check on this it came up with:

"Filter positioning is largely dependent on the complexity of the data set and the scope for which the filters are being applied (are the filters applied at the page level or only to the data set they are paired with?)."